packages/medusa/src/services/search.ts

```diff
@@ -1,13 +1,13 @@
+import { SearchService } from "medusa-interfaces"
 import { EntityManager } from "typeorm"
-import { AbstractSearchService } from "../interfaces/search-service"
 import { Logger } from "../types/global"
 
 type InjectedDependencies = {
   logger: Logger
   manager: EntityManager
 }
 
-export default class DefaultSearchService extends AbstractSearchService {
+export default class DefaultSearchService extends SearchService {
```

Comment:
suggestion: any reason to not extends SearchUtils.AbstractSearchService?

Reply:
Either we use `SearchService` or we change the compiler options of `@medusjs/utils` to es5. I see Carlos has already done the latter, so I'll do the same and use the `AbstractServiceService` )